### DISCUSSION TITLE
Train, Validation, Test performance

### DISCUSSION THREAD

**Discussion by leoin841023** [autogluon/autogluon#1360] [Q&A] [answered]
HI,
just wanna make sure,
if I want model performance on training set, validation set, and test set,

the code i use below

`predictor = TabularPredictor(label='MO24',eval_metric= 'pearsonr').fit(MO24_train, ag_args_fit={'num_gpus': 1},auto_stack=True)               

v=predictor.leaderboard(silent=True)

tr=predictor.evaluate(MO24_train)                                                                         

te=predictor.evaluate(MO24_test)   `

Is that right?

**Comment by Innixma** [comment, marked as answer]
Yes that is correct.
